src/index/coinstatsindex.cpp

```diff
@@ -122,9 +122,12 @@ bool CoinStatsIndex::WriteBlock(const CBlock& block, const CBlockIndex* pindex)
 
         uint256 expected_block_hash{pindex->pprev->GetBlockHash()};
         if (read_out.first != expected_block_hash) {
+            LogPrintf("%s: WARNING: previous block header belongs to unexpected block %s; expected %s\n",
+                      __func__, read_out.first.ToString(), expected_block_hash.ToString());
```

Comment:
```suggestion
                      read_out.first.ToString(), expected_block_hash.ToString());
```

This is toggled with `-logsourcelocations`, so can skip here.

Reply:
done